modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -96,4 +96,4 @@ $ oc get vmi vm-fedora -ojsonpath="{ @.status.interfaces }"
   }
 ]
 ----
-<1> The hot-plugged interface appears in the VMI status.
\ No newline at end of file
+<1> The hot plugged interface appears in the VMI status.
```

Comment:
I would leave the hyphen here. There is no consensus on line, but it seems to be correct English.

Reply:
I agree with you but IBM Style Guide says no hyphen for 'hot plug'.